bin/ch-run.c

```diff
@@ -51,6 +51,7 @@ const struct argp_option options[] = {
    { "join-pid",     -5, "PID",  0, "join a namespace using a PID" },
    { "join-ct",      -3, "N",    0, "number of ch-run peers (implies --join)" },
    { "join-tag",     -4, "TAG",  0, "label for peer group (implies --join)" },
+   { "no-expand",   -10, 0,      0, "don't expand $ in --set-env input"},
```

Comment:
I'd call this `--set-env-no-expand`, or something else that ties it to `--set-env`.